assets/app/view/game/game_info.rb

```diff
@@ -63,18 +72,32 @@ def phases
             },
           }
 
-          can_buy_companies = phase[:buy_companies] ? 'Can Buy Companies' : ''
-
           h(:tr, [
             h(:td, (current_phase == phase ? '→ ' : '') + phase[:name]),
             h(:td, phase[:on]),
             h(:td, phase[:operating_rounds]),
             h(:td, phase[:train_limit]),
             h(:td, phase_props, phase_color.capitalize),
-            h(:td, can_buy_companies),
+            h(:td, row_events.map(&:first).join(', ')),
           ])
         end
 
+        status_text = phases_events.uniq.map do |short, long|
+          h(:tr, [h(:td, short), h(:td, long)])
+        end
+
+        if status_text.any?
+          status_text = [h(:table, [
```

Comment:
there's no point to making this an array because you flatten it anyways on line 118

Reply:
ah i see it's because you're using status_text previously and relying on this being empty, that's a bit tricky

Reply:
I must confess that this is more or less copy-paste from the Events presentation in another place in this file.